examples/src/RecordMovieComponent.tsx

```diff
@@ -9,17 +9,15 @@ const RecordMovieComponent = ({
     isRecording,
     setIsRecording,
 }: RecordMovieComponentProps) => {
-    let [recordingDuration, setRecordingDuration] = useState<number>(0);
-    let [recordingStatus, setRecordingStatus] =
-        useState<string>("Not recording");
+    let [recordingTimeElapsed, setRecordingTimeElapsed] = useState<number>(0);
     let [outputStatus, setOutputStatus] = useState<string>("");
 
     // this useEffect is a timer that updates the recording duration
     useEffect(() => {
         let intervalId;
         if (isRecording) {
             intervalId = setInterval(() => {
-                setRecordingDuration((prevTimer) => prevTimer + 1);
+                setRecordingTimeElapsed((prevTimer) => prevTimer + 1);
```

Comment:
One more nit on the name, if it has units (looks like seconds here?) it's a good idea to document it through a comment or the variable name itself.

Reply:
Ok! Adding a comment.